src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,10 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        final int numberOfBallsToPrint = 3;
```

Comment:
You should  make  constant like:
private static final int NUMBER = 100; (Lottery::6)

Reply:
got u!